src/bin/tikv-ctl.rs

```diff
@@ -592,6 +598,18 @@ impl DebugExecutor for Debugger {
         }
         println!("all regions are healthy")
     }
+
+    fn unsafe_conf_change(&self, conf_change: UnsafeConfChange) {
+        let region_errors = self.unsafe_conf_change(conf_change)
```

Comment:
This is confusing. Generally, error should be put into the `E` part of `Result<T, E>`.